CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -293,5 +294,62 @@ def __init__(self, db_client: "botocore.client.DynamoDB"):
 
 
 class Tag(DB_Item):
-    def __init__(self, db_client: "botocore.client.DynamoDB"):
+    REQUIRED_FIELDS = ['tag_name']
+    TABLE_NAME = "tag"
+
+    def __init__(self, db_client: "botocore.client.DynamoDB", tag_name:str= None):
         super().__init__(db_client=db_client)
+        self.tag_name = self.format_tag(tag_name)
+
+    def format_tag(self, tag_name:str) -> str:
+        #Tag_name is alphanumeric, can include hyphen and underscore, with no spaces and all lower cased
+        if tag_name:
+            tag_name = tag_name.lower().replace(" ", "")
+        return tag_name
+
+    def is_valid_tag_name(self, tag_name:str) -> Tuple[bool, str]:
```

Comment:
This method should belong to the class, since it isn’t instance specific. Can we make this a static method on the Tag class?